src/ImageSharp/Formats/Jpeg/Components/Decoder/SpectralConverter{TPixel}.cs

```diff
@@ -65,26 +62,26 @@ public override void InjectFrameData(JpegFrame frame, IRawJpegData jpegData)
             MemoryAllocator allocator = this.configuration.MemoryAllocator;
 
             // iteration data
-            IJpegComponent c0 = frame.Components[0];
+            int majorBlockWidth = frame.Components.Max((component) => component.SizeInBlocks.Width);
```

Comment:
How’s the perf using Max here? We’ve avoided Linq everywhere in the codecs AFAIK

Reply:
Considering it's from 1 to 4 elements and it executes only once I don't think it will matter but I will run benchmarks just to be sure.